MAINTAINERS

```diff
@@ -591,7 +598,7 @@ made through a pull request.
 
 	[people.jfrazelle]
 	Name = "Jessie Frazelle"
-	Email = "jess@docker.com"
+	Email = "jess@linux.com"
```

Comment:
I'm absolutely fine leaving it for another discussion, because I'm LGTM to the addition; security is important and I think @diogomonica and @NathanMcCauley will be a wonderful addition to the security team here.